doc/releases/release-notes-4.1.rst

```diff
@@ -24,359 +47,178 @@ https://docs.zephyrproject.org/latest/security/vulnerabilities.html
+   to use POSIX API socket calls, one needs to enable the
+   :kconfig:option:`CONFIG_POSIX_API` option.  If the application does not want
+   or is not able to enable that option, then the socket API calls need to be
+   prefixed by a ``zsock_`` string.
 
-* native/POSIX
+Deprecated APIs
+===============
 
-  * :kconfig:option:`CONFIG_NATIVE_APPLICATION` has been deprecated.
-  * For the native_sim target :kconfig:option:`CONFIG_NATIVE_SIM_NATIVE_POSIX_COMPAT` has been
-    switched to ``n`` by default, and this option has been deprecated.
+* the :c:func:`bt_le_set_auto_conn` API function. Application developers can achieve
+  the same functionality in their application code by reconnecting to the peer when the
+  :c:member:`bt_conn_cb.disconnected` callback is invoked.
 
-Kernel
-******
+* :kconfig:option:`CONFIG_NATIVE_APPLICATION` has been deprecated.
 
-Bluetooth
-*********
+* For the native_sim target :kconfig:option:`CONFIG_NATIVE_SIM_NATIVE_POSIX_COMPAT` has been
+  switched to ``n`` by default, and this option has been deprecated.
 
-* Audio
+* :kconfig:option:`CONFIG_BT_BUF_ACL_RX_COUNT`
 
-* Host
+* All HWMv1 board name aliases which were added as deprecated in v3.7 are now removed
+  (:github:`82247`).
 
-  * :kconfig:option:`CONFIG_BT_BUF_ACL_RX_COUNT` has been deprecated and
-    :kconfig:option:`CONFIG_BT_BUF_ACL_RX_COUNT_EXTRA` has been added.
+New APIs and options
+====================
 
-* HCI Drivers
+..
+  Link to new APIs here, in a group if you think it's necessary, no need to get
+  fancy just list the link, that should contain the documentation. If you feel
+  like you need to add more details, add them in the API documentation code
+  instead.
 
-Boards & SoC Support
-********************
+* Architectures
 
-* Added support for these SoC series:
+  * :kconfig:option:`CONFIG_ARCH_HAS_CUSTOM_CURRENT_IMPL`
+  * :kconfig:option:`CONFIG_RISCV_CURRENT_VIA_GP`
 
-* Made these changes in other SoC series:
+* Crypto
 
-* Added support for these boards:
+  * :kconfig:option:`CONFIG_MBEDTLS_PSA_STATIC_KEY_SLOTS`
+  * :kconfig:option:`CONFIG_MBEDTLS_PSA_KEY_SLOT_COUNT`
+
+* Other
+
+  * :kconfig:option:`CONFIG_BT_BUF_ACL_RX_COUNT_EXTRA`
+  * :c:macro:`DT_ANY_INST_HAS_BOOL_STATUS_OKAY`
+  * :c:struct:`led_dt_spec`
+
+New Boards & SoC
+****************
+..
+  Just link the board, details go in the board description
+
+* :zephyr:board:`max78002evkit`
+* :zephyr:board:`acp_6_0_adsp`
+* :zephyr:board:`ucan`
+* :zephyr:board:`ttgo_t7v1_5`
+* :zephyr:board:`ttgo_t8s3`
+* :zephyr:board:`m5stack_cores3`
+* :zephyr:board:`mks_canable_v20`
+* :zephyr:board:`octopus_io_board`
+* :zephyr:board:`octopus_som`
+* :zephyr:board:`candlelight`
+* :zephyr:board:`candlelightfd`
+* :zephyr:board:`xiao_esp32c6`
+* :zephyr:board:`rp2040_zero`
+* :zephyr:board:`ch32v003evt`
+* :zephyr:board:`we_orthosie1ev`
+* :zephyr:board:`mini_stm32h7b0`
+
+New Drivers
+***********
+..
+  Just link the driver, details go in the binding description
```

Comment:
Assuming we'll do the same here: 

```suggestion
    Same as above for boards, this will also be recomputed at the time of the release.
    Just link the driver, details go in the binding description
```

Reply:
done